src/applications/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -112,22 +118,45 @@ const RatedDisabilityList = ({
     );
   }
 
-  const formattedDisabilities = formalizeData(
+  const serviceConnected = getServiceConnectedDisabilities(
+    ratedDisabilities?.ratedDisabilities,
+  );
+
+  const nonServiceConnected = getNonServiceConnectedDisabilities(
     ratedDisabilities?.ratedDisabilities,
-  ).sort(sortFunc);
+  );
+
+  const formattedServiceConnected = formalizeData(serviceConnected).sort(
+    sortFunc,
+  );
+
+  const formattedNonServiceConnected = formalizeData(nonServiceConnected).sort(
+    sortFunc,
+  );
 
   return (
     <div>
-      <h2 id="individual-ratings" className="vads-u-margin-y--1p5">
-        Your individual ratings
-      </h2>
+      <h3 className="vads-u-margin-y--2">Service-connected ratings</h3>
+      {sortToggle && (
+        <div id="ratings-sort-select-ab" className="vads-u-margin-bottom--2">
+          <SortSelect onSelect={setSortBy} sortBy={sortBy} />
+        </div>
```

Comment:
I see this same code on lines 153-156; does that mean when you sort one, you sort both?

Reply:
I may just need to remove the second one. This is behind a feature toggle that has never been turned on and I think that we probably just need to revisit the question of if there is any value in adding a sorting option here